app/controllers/admin/departments_controller.rb

```diff
@@ -17,7 +17,7 @@ class DepartmentsController < Admin::ApplicationController
     # for more information
 
     def find_resource(param)
-      Department.find_by! slug: param
+      Department.find param
```

Comment:
I do however think there's value in maintaining URL structure when context-switching, so that an author/reviewer can go from `/foo/bar` to `/editorial/foo/bar` and have it immediately obvious from the URL structure where they are in the hierarchy of the site. Agreed we shouldn't keep it just because we had it, but we do need to question is there value in preserving this for editorial folk.